### PR TITLE
docs: update the new structure and landing page

### PR DESCRIPTION
**Pull request by yuxizama**
- Added new landing page and designs
- Reorganized existing contents

### CODE REVIEW

**Comment by RomanBredehoft** [comment]
Thanks @yuxizama ! It looks like several images are not used (anymore ?) with your changes : 
```
Image is not used: docs/.gitbook/assets/fundamentals.png
Image is not used: docs/.gitbook/assets/demo_gpt2.png
Image is not used: docs/.gitbook/assets/image_9.png
Image is not used: docs/.gitbook/assets/demo_filtering.png
Image is not used: docs/.gitbook/assets/guides.png
Image is not used: dark.png
```

File: docs/getting-started/readme.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
+# What is Concrete ML?
+
+[⭐️ Star the repo on Github](https://github.com/zama-ai/concrete-ml) | [🗣 Community support forum](https://community.zama.ai/c/concrete-ml/8) | [📁 Contribute to the project](../developer/contributing.md)
+
+![](<../.gitbook/assets/3%20(1)%20(1).png>)
```
Comment (RomanBredehoft) [inline docs/getting-started/readme.md:5]: this image does not exist in the PR !

File: src/concrete/ml/common/utils.py [hunk 2]
```diff
@@ -38,7 +38,7 @@
 # when simulating FHE executions
 # Set 'USE_OLD_VL' to False by default once the new simulation is fixed
 # FIXME: https://github.com/zama-ai/concrete-ml-internal/issues/4091
-USE_OLD_VL = True
+USE_OLD_VL = False
```
Comment (andrei-stoian-zama) [inline src/concrete/ml/common/utils.py:41]: @RomanBredehoft what's going on here :) ?

Comment (RomanBredehoft) [reply src/concrete/ml/common/utils.py:41]: oups 🙃

**Pull request merged**